code/game/objects/structures/watercloset.dm

```diff
@@ -478,3 +478,45 @@
 	icon_state = "puddle-splash"
 	. = ..()
 	icon_state = "puddle"
+
+
+//Shower Curtains//
+
+#define SHOWER_OPEN_LAYER OBJ_LAYER + 0.4
+#define SHOWER_CLOSED_LAYER MOB_LAYER + 0.2
```

Comment:
Reusing defines means you change more stuff than you want to, is that a wise thing to do?